### DISCUSSION TITLE
DDR Imgorarg's loyalty

### DISCUSSION THREAD

**Discussion by FrozenEmpire** [ChilliHugger/The-Lords-Of-Midnight#197] [Code Algorithm Questions] [answered]
Hi Chris,

I noticed that CH_IMGORARG starts with a loyalty to RA_DEFAULT. I think this should be RA_DWARF.
(No other characters start with RA_DEFAULT...)

I noticed this because CH_TORORMARG (whose liege is CH_IMGORARG), recruited CH_IMGORARG since he was not of the same loyalty :)

Regards
Andrew

**Comment by IcemarkUK** [comment, marked as answer]
Yep! Definitely a problem.
